### ISSUE TITLE
Ouput log of training fcn-32s and fcn-16s

### ISSUE THREAD

**Issue by fangyizhang** [dmlc/mxnet#2160]
Recently, I am reproducing the result of fcn-xs in the mxnet example, I exactly follow the instruction from the README.md file, but the fcn-8s output is not like below, my train-accuracy is around 0.6 to 0.8, never get to 0.9. 

the output log may like this(when training fcn-8s):
INFO:root:Start training with gpu(3)
INFO:root:Epoch[0] Batch [50]   Speed: 1.16 samples/sec Train-accuracy=0.894318
INFO:root:Epoch[0] Batch [100]  Speed: 1.11 samples/sec Train-accuracy=0.904681
INFO:root:Epoch[0] Batch [150]  Speed: 1.13 samples/sec Train-accuracy=0.908053
INFO:root:Epoch[0] Batch [200]  Speed: 1.12 samples/sec Train-accuracy=0.912219
INFO:root:Epoch[0] Batch [250]  Speed: 1.13 samples/sec Train-accuracy=0.914238
INFO:root:Epoch[0] Batch [300]  Speed: 1.13 samples/sec Train-accuracy=0.912170
INFO:root:Epoch[0] Batch [350]  Speed: 1.12 samples/sec Train-accuracy=0.912080

Here is what I do.
First I train the fcn-32s, change the epoch: 31
Then I train the fcn-16s, change the learning rate: 10e-12 and epoch: 27
Finally I train the fcn-8s, change the learning rate: 10e-14 and epoch:19

Can anyone tell me the output log of fcn-32s and fcn-16s, just the end or the beginning log will be ok, so I can figure out which training stage is wrong

**Comment by tornadomeet** [comment]
you should use larger lr(not the lr in the readme file ), because of  softmaxoutput operator has been changed from then on.   
or you can delete the line https://github.com/dmlc/mxnet/blob/master/src/operator/softmax_output-inl.h#L108, and rebuild mxnet,then use the lr describled in the readme.

**Comment by fangyizhang** [comment]
@tornadomeet Thanks! I am not familiar with the source code. Could you tell me grad_scale and s[3] mean in https://github.com/dmlc/mxnet/blob/master/src/operator/softmax_output-inl.h#L108.

**Comment by tornadomeet** [comment]
@fangyizhang  the line is used for gradient normalization, s3[2] is the image label size(equals image size,and it is very large), so if we do not divide this,then the gradient will be very large(for it sums all the label), so we need small lr,like 1e-10.  
if we divide the label numbers in that line, then the gradient is normalized, so we can use normal lr, which is larger than the lr in the readme.

**Comment by fangyizhang** [comment]
@tornadomeet Thanks a lot. I delete the code, then it works.
